### PR TITLE
adding initial opensearch-tunnel module

### PR DESCRIPTION
**Pull request by dgraeber**
*Issue #, if available:*

*Description of changes:*
Initial openseach-tunnel...without unit tests or gitworkflows for now....(to be added shortly)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: modules/integration/opensearch-tunnel/deployspec.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+deploy:
+  phases:
+    install:
+      commands:
+      - npm install -g aws-cdk@2.82.0
+      - pip install -r requirements.txt
+    build:
+      commands:
+      - cdk deploy --require-approval never --progress events --app "python app.py" --outputs-file ./cdk-exports.json
+      #- export ADDF_MODULE_METADATA=$(python -c "import json; file=open('cdk-exports.json'); print(json.load(file)['addf-${ADDF_DEPLOYMENT_NAME}-${ADDF_MODULE_NAME}']['metadata'])")
```
Comment (srinivasreddych) [inline modules/integration/opensearch-tunnel/deployspec.yaml:10]: Can you delete this line?

File: modules/integration/opensearch-tunnel/install_nginx.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```
Comment (srinivasreddych) [inline modules/integration/opensearch-tunnel/install_nginx.sh:1]: Can you delete this file?

Comment (dgraeber) [reply modules/integration/opensearch-tunnel/install_nginx.sh:1]: Noooooo

File: modules/integration/opensearch-tunnel/deployspec.yaml (hunk 1 above)
Comment (dgraeber) [reply modules/integration/opensearch-tunnel/deployspec.yaml]: reverted to use old method for now

File: modules/integration/opensearch-tunnel/install_nginx.sh (hunk 2 above)
Comment (dgraeber) [reply modules/integration/opensearch-tunnel/install_nginx.sh:1]: this is required for the install

**Pull request merged**